### PR TITLE
Update configure-automated-investigations-remediation.md

### LINKED ISSUE #11517: Documentation is not updated with the latest changes
[Enter feedback here]

Based on this [link](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Ftechcommunity.microsoft.com%2Ft5%2Fmicrosoft-defender-for-endpoint%2Fmicrosoft-defender-for-endpoint-automation-defaults-are-changing%2Fba-p%2F2068744&data=05%7C01%7Cfshehadeh%40microsoft.com%7C66d3e4b4f9d542af0e9908db2f43696a%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638155739108529904%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ZkAJbhXBfVztXjTjwi%2FaJXM7XffLj1gK%2FlElJ1qCyUE%3D&reserved=0) and PMs confirmation, Automated Investigation option has been removed from Advanced features setting and it's enabled by default for all users

However, official documentation showing that to enable this feature we should turn it on from the settings:
image

Please update this

Thanks,
Fatima

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 6e0afc1f-2112-050d-2db6-40b14c2149b9
* Version Independent ID: 559af624-9032-7d66-670b-081ce8ac9d44
* Content: [Configure automated investigation and remediation capabilities](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation?view=o365-worldwide#turn-on-automated-investigation-and-remediation)
* Content Source: [microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that the feature is removed from advanced features and enabled by default to all users

fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11517

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md [hunk 1]
```diff
@@ -40,15 +40,9 @@ To configure automated investigation and remediation:
...
 
-1. As a global administrator or security administrator, go to the Microsoft 365 Defender portal ([https://security.microsoft.com](https://security.microsoft.com)) and sign in.
-
-2. In the navigation pane, choose **Settings**.
-
-3. Select **Endpoints**, then select **Advanced features**.
-
-4. Turn on both **Automated Investigation** and **Automatically resolve alerts**.
+> [!NOTE]
+> Automated Investigation option has been removed from Advanced features setting and it's enabled by default for all users.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation.md]:
```suggestion
> The Automated Investigation option has been removed from the Advanced features setting and is enabled by default for all users.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by denisebmsft — APPROVED** [review]
Thank you @msbemba @JohanFreelancer9

**Pull request merged**